package.json

```diff
@@ -56,68 +56,68 @@
     "styled-components": ">= 5.3.0"
   },
   "dependencies": {
-    "@ant-design/icons": "^4.7.0",
-    "@react-aria/button": "3.6.2",
-    "@react-aria/checkbox": "3.6.0",
-    "@react-aria/combobox": "3.4.2",
-    "@react-aria/dialog": "3.4.0",
-    "@react-aria/focus": "3.9.0",
-    "@react-aria/i18n": "3.6.1",
-    "@react-aria/interactions": "3.12.0",
-    "@react-aria/listbox": "3.7.0",
-    "@react-aria/menu": "3.6.2",
-    "@react-aria/numberfield": "3.3.2",
-    "@react-aria/overlays": "3.11.0",
-    "@react-aria/radio": "3.4.0",
-    "@react-aria/searchfield": "3.4.4",
-    "@react-aria/select": "3.8.2",
-    "@react-aria/selection": "3.11.0",
-    "@react-aria/separator": "3.2.4",
-    "@react-aria/slider": "3.2.4",
-    "@react-aria/ssr": "3.3.0",
-    "@react-aria/switch": "3.2.4",
-    "@react-aria/textfield": "3.7.2",
-    "@react-aria/tooltip": "3.3.2",
-    "@react-aria/utils": "3.14.0",
-    "@react-aria/visually-hidden": "3.5.0",
-    "@react-spectrum/utils": "3.7.4",
-    "@react-stately/checkbox": "3.3.0",
-    "@react-stately/collections": "3.4.4",
-    "@react-stately/combobox": "3.2.2",
-    "@react-stately/list": "3.5.4",
-    "@react-stately/menu": "3.4.2",
-    "@react-stately/numberfield": "3.2.2",
-    "@react-stately/overlays": "3.4.2",
-    "@react-stately/radio": "3.6.0",
-    "@react-stately/searchfield": "3.3.4",
-    "@react-stately/select": "3.3.2",
+    "@ant-design/icons": "^5.0.1",
+    "@react-aria/button": "^3.6.4",
+    "@react-aria/checkbox": "^3.7.1",
+    "@react-aria/combobox": "^3.4.4",
+    "@react-aria/dialog": "^3.4.2",
+    "@react-aria/focus": "^3.10.1",
+    "@react-aria/i18n": "^3.6.3",
+    "@react-aria/interactions": "^3.13.1",
+    "@react-aria/listbox": "^3.7.2",
+    "@react-aria/menu": "^3.7.1",
+    "@react-aria/numberfield": "^3.3.4",
+    "@react-aria/overlays": "^3.12.1",
+    "@react-aria/radio": "^3.4.2",
+    "@react-aria/searchfield": "^3.4.4",
+    "@react-aria/select": "^3.8.4",
+    "@react-aria/selection": "^3.12.1",
+    "@react-aria/separator": "^3.2.6",
+    "@react-aria/slider": "^3.2.4",
+    "@react-aria/ssr": "^3.4.1",
+    "@react-aria/switch": "^3.3.1",
+    "@react-aria/textfield": "^3.8.1",
+    "@react-aria/tooltip": "^3.3.4",
+    "@react-aria/utils": "^3.14.2",
+    "@react-aria/visually-hidden": "^3.6.1",
```

Comment:
Please, don't use `^` in react-aria, react-stately, and react-types packages, react-spectrum doesn't handle well differences between packages. We need a 100% deterministic install process for us and our consumers

Reply:
@MrFlashAccount fixed